atest/robot/libdoc/toc_generation_lib_init.robot

```diff
@@ -0,0 +1,61 @@
+*** Settings ***
+Suite Setup       Run Libdoc And Parse Output    ${TESTDATADIR}/TocLibraryInitRobot.py
+
+Resource          libdoc_resource.robot
+
+*** Test Cases ***
+Name
+    Name Should Be                  TocLibraryInitRobot
+
+Documentation
+    Doc Should Be
+    ...    Simple library
+    ...
+    ...    == Table of contents ==
+    ...
+    ...    - `1 Heading`
+    ...    - `2 Heading`
+    ...    - `Importing`
+    ...    - `Shortcuts`
+    ...    - `Keywords`
+    ...
+    ...    = 1 Heading =
+    ...
+    ...    Some text here for chapter 1.
+    ...
+    ...    = 2 Heading =
+    ...
+    ...    Second chapter text there.
+    ...
+    ...    Second paragraph.
+
+Version
+    Version Should Be               ${EMPTY}
+
```

Comment:
Do we actually need to test other information from the library than the documentation? If we'd only check that, there could be just one test case file that could have separate tests for lib w/ and w/o toc, resource file, etc.

Reply:
Well, perhaps not. I did not read that code so deeply and just took examples from other libdoc tests.